### PR TITLE
[Fixes #3401] ourNews: user with no role with access to post

### LINKED ISSUE #3401: ourNews: user with no role with access to post
should user with no role can post in ourNews?

### PR DESCRIPTION
**Pull request by coder8102**
[Fixes #3401] ourNews: user with no role with access to post

**Learner**
issue#3401_learner

**No role**
issue#3401_no-role

### CODE REVIEW

**Comment by coder8102** [comment]
@paulbert How to allow leader and manager having access to POST? Should I just add roles into condition as *planetAuthorizedRoles="'learner',''leader','manager'"

File: src/app/news/news.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <mat-toolbar-row class="primary-color font-size-1"></mat-toolbar-row>
   </mat-toolbar>
   <div class="view-container view-full-height">
-    <div class="new-post-form">
+    <div *planetAuthorizedRoles="'learner'" class="new-post-form">
```
Comment (yuviii) [inline src/app/news/news.component.html]: @coder8102 `<div *ngIf="currentUser.isUserAdmin || currentUser.roles.length" class="news-post-form">` will work for this case.

**Comment by paulbert** [comment]
@yuviii That is the way we would do this before, but we should use the custom `*planetAuthorizedRoles` directive instead of `*ngIf`.

Also I'll add an issue related to my message in the gitter channel:
```
As far as leaders & managers not having access to post, we need to make a separate change to how roles are set to fix this. Every activated member should be a learner at a minimum, so setting *planetAuthorizedRoles="'learner'" on an element will show it to every activated member (members can be deactivated by clicking the lock symbol on the Members page)
```

**Comment by paulbert** [comment]
See #3488 as follow up to make this work for leaders as well.  Leader should also be learner always (so we should not allow members with only leader role)

**Pull request merged**